src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -502,7 +502,7 @@ namespace SurfaceGeometry {
 
         for (SurfNum = 1; SurfNum <= TotSurfaces; ++SurfNum) { // Loop through all surfaces to find windows...
 
-            if (!Surface(SurfNum).HeatTransSurf) continue; // Skip shadowing (sub)surfaces
+            if (!Surface(SurfNum).HeatTransSurf && !Surface(SurfNum).IsAirBoundarySurf) continue; // Skip shadowing (sub)surfaces
```

Comment:
This lets air boundary surfaces be included in things like the average ceiling height calculation (and avoids a div by zero if the only ceiling surface is an air boundary surface).